src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let copyState = { ...state };
```

Comment:
Consider using 'const' instead of 'let' for 'copyState' since it is never re-assigned directly.